vernac/vernacentries.mli

```diff
@@ -19,12 +19,13 @@ val check_may_eval :
 val translate_vernac
   : ?loc:Loc.t
   -> atts:Attributes.vernac_flags
-  -> Vernacexpr.vernac_expr
+  -> Synterp.vernac_entry
   -> Vernacextend.typed_vernac
 
 (** Vernacular require command, used by the command line *)
 val vernac_require
-  : Libnames.qualid option
+  : Library.library_t list
+  -> Names.DirPath.t list
```

Comment:
Maybe we should keep this as the combined version and add `vernac_interp_require` if useful?

Reply:
I guess the split version would be useful for `coqc -only-parsing -require foo`

Reply:
Done.